Userland/Utilities/keymap.cpp

```diff
@@ -49,13 +49,26 @@ int main(int argc, char** argv)
 
     const char* path = nullptr;
     Core::ArgsParser args_parser;
-    args_parser.add_positional_argument(path, "The mapping file to be used", "file");
+    args_parser.add_positional_argument(path, "The mapping file to be used", "file", Core::ArgsParser::Required::No);
     args_parser.parse(argc, argv);
 
+    dbgln("path is at {:p}, and contains {}", path, path);
+
+    if (!path) {
+        auto keymap = Keyboard::CharacterMap::fetch_system_map();
+        if (!keymap.has_value()) {
+            perror("getkeymap");
```

Comment:
Please don't use `errno` to carry errors outside `LibC`. Instead, return a `Result<T, OSError>` or similar.